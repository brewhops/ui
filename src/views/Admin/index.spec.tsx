import React from 'react';
import { Admin } from '../Admin';
import { shallow } from 'enzyme';

describe('src/pages/Admin/index.tsx', () => {
  it('renders the Admin page', () => {
    expect(shallow(<Admin/>)).toMatchSnapshot();
  })
});