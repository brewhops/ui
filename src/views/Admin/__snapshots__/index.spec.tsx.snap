// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/Admin/index.tsx renders the Admin page 1`] = `ShallowWrapper {}`;
